<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOrgan</class>
 <widget class="QDialog" name="DialogOrgan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>组织管理能力</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QComboBox" name="cbB_PositionClass"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>干部考核</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>职位类型</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pbt_Ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="icon">
        <iconset resource="iamges.qrc">
         <normaloff>:/images/704.bmp</normaloff>:/images/704.bmp</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbB_TLevel"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>修改</string>
       </property>
       <property name="icon">
        <iconset resource="iamges.qrc">
         <normaloff>:/images/804.bmp</normaloff>:/images/804.bmp</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>职位名称</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>641</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>职位类型：</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>621</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>第一级：校团委学生副书记、校学生会主席团成员、团总支副书记、学生分会副主席、学生党支部副书记（支部委员）等。</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>第二级：校团委、校学生会、团总支、学生分会等部门负责人、学生党支部党小组组长、学生行政班班长和团支部书记、园区长及园区部长、新生班主任及大班长等。</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>第三级：校团委、校学生会、团总支、学生分会等部门干事、学生行政班班委和团支部支委委员、园区楼栋长和楼层长、学生社团负责人等。</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>第四级：学生寝室室长、实习小组组长、学生社团理事等。</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbt_Cancel">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>250</y>
     <width>136</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="icon">
    <iconset resource="iamges.qrc">
     <normaloff>:/images/706.bmp</normaloff>:/images/706.bmp</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="iamges.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbt_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogOrgan</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>pbt_Ok</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
